websites/H/Hoppscotch/presence.ts

```diff
@@ -0,0 +1,51 @@
+import { GraphQL } from "./routes/graphql";
+import { MQTT } from "./routes/mqtt";
+import { Rest } from "./routes/rest";
+import { Settings } from "./routes/settings";
+import { SocketIO } from "./routes/socketio";
+import { ServerSentEvents } from "./routes/sse";
+import { Websocket } from "./routes/websocket";
+
+const presence = new Presence({
+	clientId: "792735245488488458",
+});
+
+export const enum Assets {
+	Logo = "https://hoppscotch.io/icon.png",
+	Rest = "https://i.imgur.com/CKj7VUF.png",
+	GraphQL = "https://i.imgur.com/00szkIv.png",
+	Websocket = "https://i.imgur.com/daZZiEo.png",
+	ServerSentEvents = "https://i.imgur.com/CEL3dNu.png",
+	SocketIO = "https://i.imgur.com/UvlTI6Y.png",
+	MQTT = "https://i.imgur.com/r5NFbHF.png",
+	Settings = "https://i.imgur.com/UxuTIv7.png",
```

Comment:
[Assets](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) should be 512x512px